files/default/solr-log4j.properties

```diff
@@ -0,0 +1,29 @@
+# Default Solr log4j config
+# rootLogger log level may be programmatically overridden by -Dsolr.log.level
+solr.log=${solr.log.dir}
+log4j.rootLogger=INFO, file, CONSOLE
+
+# Console appender will be programmatically disabled when Solr is started with option -Dsolr.log.muteconsole
+log4j.appender.CONSOLE=org.apache.log4j.ConsoleAppender
+log4j.appender.CONSOLE.layout=org.apache.log4j.EnhancedPatternLayout
+log4j.appender.CONSOLE.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss.SSS} %-5p (%t) [%X{collection} %X{shard} %X{replica} %X{core}] %c{1.} %m%n
+
+#- size rotation with log cleanup.
+log4j.appender.file=org.apache.log4j.RollingFileAppender
```

Comment:
Maybe leave it as single file like we have in GI and let logrotate handle it. it also does not need to restart the service (like it does need to do with apache).

heres some info on letting log4j do the role but have logrotate do the cleanup. https://serverfault.com/a/961369

gi infrastructure: /configuration/vars/app_logrotate.yml  has what you need for hourly rotation.